.adops/workflow-build.yml

```diff
@@ -30,6 +32,18 @@ stages:
                 yarn
               path: $(YARN_CACHE_FOLDER)
             displayName: Cache Yarn packages
+          - script: |
+              ORIGINAL_PKG_VER=`npx -q json -f "$(workflowPkgPath)/package.json" version`
+              NEW_PKG_VER="${ORIGINAL_PKG_VER}-$(tag)"
+
+              echo "Current package.json version is: [${ORIGINAL_PKG_VER}]"
+              echo "Updating package.json version to: [${NEW_PKG_VER}]"
+
+              npx -q json -I -f "$(workflowPkgPath)/package.json" -e "this.version = '${NEW_PKG_VER}'"
+
+              echo "Updated package.json version is: [`npx -q json -f "$(workflowPkgPath)/package.json" version`]"
```

Comment:
@EmperorRXF I think we gotta escape the quotations after `json -f` part. Otherwise it encompasses the rest of the yaml including `displayName` field.

Reply:
seems the issue is with GH's web-based syntax highlighter not being too great at parsing nested double quotes. it fails to parse even with the escapes :)

but I will still leave in the escaped quotes, as the logic on bash works nevertheless (and semantically since it's a nested string I don't mind keeping the inner ones escaped)